### ISSUE TITLE
Protected/Private 'Dispose' method should be first (non-static) protected/private method

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#424]
If there is a non-public 'Dispose' method, then it should be the first (non-static) method of that accessibility. This is because it belongs to the destruction of an object.
